eventlog/README.md

```diff
@@ -50,18 +50,18 @@ The detailed API can be found on
 ### Java API
 In java, you need to construct a `EventLogInstance` class. There are two ways
 to initialise it, the first for when you do not have an existing eventlog
-instance on omniledger to connect to, the other when you do.
+instance on ByzCoin to connect to, the other when you do.
 
 ```java
-// Create the eventlog instance. It expects an omniledger RPC, a list of 
+// Create the eventlog instance. It expects an ByzCoin RPC, a list of 
```

Comment:
`find . -name "*go" | xargs perl -pi -e "s/an Byz/a Byz/"`

Reply:
Fixed all of them in branch no_rh, coming soon.